doc/release/yarp_3_6/robotinterface_portprefix.md

```diff
@@ -0,0 +1,5 @@
+robotinterface_portprefix {#yarp_3_6}
```

Comment:
Shouldn't this be on master @elandini84 @randaz81? (yarp 3.7)
```suggestion
robotinterface_portprefix {#yarp_3_7}
```

Reply:
We can have it on yarp 3.7 (master), but I guess this will delay gazebo-yarp-plugins 5 release (see https://github.com/robotology/gazebo-yarp-plugins/issues/594) until YARP 3.7 is released.

Reply:
for me can be also merged in 3.6, @randaz81 @elandini84 do you agree?

Reply:
Since the new feature is actually fixing part of a reported issue [(#2492)](https://github.com/robotology/yarp/issues/2492) and it doesn't affect the already existing code or xml files, for me it can be merged on `yarp-3.6`.